src/contrib/mlir/backend/pass/affine_lowerer.cpp

```diff
@@ -234,7 +249,7 @@ namespace
         // Create target that defines legal ops for nGraph dialect to be lowered to.
         ConversionTarget target(getContext());
 
-        target.addLegalDialect<AffineOpsDialect, StandardOpsDialect>();
+        target.addLegalDialect<AffineOpsDialect, StandardOpsDialect, LLVM::LLVMDialect>();
```

Comment:
Do we need LLVM dialect ? You are emitting std calls, right ?

Reply:
will remove